.github/workflows/_linux-build.yml

```diff
@@ -199,7 +199,8 @@ jobs:
           SHA1: ${{ github.event.pull_request.head.sha || github.sha }}
           SCCACHE_BUCKET: ossci-compiler-cache-circleci-v2
           SCCACHE_REGION: us-east-1
-          SCCACHE_S3_KEY_PREFIX: ${{ github.workflow }}
+          # SCCACHE_S3_KEY_PREFIX: ${{ github.workflow }}
```

Comment:
this is still from testing? We don;t want to put everything in debug/, right?

Reply:
Yeah, this is just temp change to test the PR, I'm reverting it before landing.  I want to use a fresh empty cache here for a clean build as it will hit the compilation of flash attention.  If the memory swap didn't work, some jobs would fail there.

Reply:
Signals looks green, so I'm reverting this change now